File: ppmtool/src/main/java/com/lige110/springreact/ppmtool/PpmtoolApplication.java
package com.lige110.springreact.ppmtool;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;

@SpringBootApplication(exclude = ErrorMvcAutoConfiguration.class)
public class PpmtoolApplication {

    public static void main(String[] args) {
        SpringApplication.run(PpmtoolApplication.class, args);
    }

}

File: ppmtool/src/main/kotlin/com/lige110/ppmtool/PpmtoolApplication.kt
package com.lige110.ppmtool

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(exclude = [ErrorMvcAutoConfiguration::class])
class PpmtoolApplication

fun main(args: Array<String>) {
    runApplication<PpmtoolApplication>(*args)
}
